libs/NfcCoreLib/lib/Nci/phNciNfc.h

```diff
@@ -287,6 +287,24 @@ typedef enum phNciNfc_NfceeModes
    PH_NCINFC_NFCEEDISC_UNKNOWN
 }phNciNfc_NfceeModes_t;
 
+/**
+* \ingroup grp_nci_nfc
+* \brief NFCEE Power and Link Control Info contains power related identifier..
+*/
+typedef enum phNciNfc_PowerLinkMode
+{
+    /** NFCEE  NFCC decides identifier*/
```

Comment:
nit: Place comments on the same line as the value, for consistency with the reset of the file.

Reply:
I have noted a couple of enumeration like that at least in this file. Not sure what the rule is
I have aligned comments and enum values only for phNciNfc_PowerLinkMode.